web_src/less/themes/theme-arc-green.less

```diff
@@ -531,15 +515,31 @@ a.ui.basic.green.label:hover {
     border-bottom: 1px solid #304251;
 }
 
-.repository.wiki.new .ui.container form .ui.tabular.menu {
-    border-bottom: 1px solid rgba(187, 187, 187, .6);
+.ui.tabular.menu {
+    border-bottom: 1px solid rgba(187, 187, 187, .24);
+
+    .item.active {
+        border-top: 1px solid transparent;
+        border-left: 1px solid transparent;
+        border-right: 1px solid transparent;
```

Comment:
Do those need the full shorthands? If the width is already present in the base style, I suggest just setting `border-color`.